lc-hax/Scripts/Modules/ESPMod.cs

```diff
@@ -31,7 +30,7 @@ public class ESPMod : MonoBehaviour {
         if (!this.InGame || !Helper.CurrentCamera.IsNotNull(out Camera camera)) return;
 
         this.PlayerRenderers.ForEach(rendererPair => {
-            if (rendererPair.GameObject.isPlayerDead) return;
+            if (rendererPair.GameObject.Unfake()?.isPlayerDead ?? true) return;
```

Comment:
Is player somehow becoming null?

Reply:
just a precaution because of the null exceptions with the other renderers, I assume it could happen during disconnect

Reply:
I don't think so. When a player disconnect, their player object just returns to the void. They aren't destroyed.